docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurator.java

```diff
@@ -87,6 +96,54 @@ private void configure(ArquillianDescriptor arquillianDescriptor) {
         return cubeConfiguration;
     }
 
+    private Map<String, String> resolveDownloadDockerMachine(Map<String, String> config) {
+        if (config.containsKey(CubeDockerConfiguration.DOCKER_MACHINE_NAME)) {
+            String machineVersion = GitHubUtil.getDockerMachineLatestVersion();
```

Comment:
It seems like you will _always_ communicate with GitHub as long as machineName is defined. This needs to be reordered to check if 'docker-machine' exists on path before anything else. Then if it does exist already, move on without doing anything else.